### PR TITLE
Zipcode

### PR DESCRIPTION
**Pull request by djc00p**
## Pull Request Review Template

- [x] Wrote Tests
- [x] Implemented
- [] Reviewed

### Necessary checkmarks:
- [x] All Tests are Passing
- [x] The code will run locally

### Type of change
- [] New feature
- [] Bug Fix

### Implements/Fixes:
Zip code Api

### Description of Changes:
Accessing the Zip Code Api but it does only have a 10 hit per hour limit so may need to re think use.

closes#24

### Check the correct boxes
- [x] This broke nothing
- [] This broke some stuff
- [] This broke everything

### Testing Changes
- COVERAGE SCORE:
- [x] New Tests have been added
- [] No Tests have been changed
- [x] Some Tests have been changed
- [] All of the Tests have been changed (Please describe what in the world happened)

### Checklist:
- [x] My code has no unused/commented out code
- [] I have reviewed my code
- [] I have commented my code, particularly in hard-to-understand areas
- [] I have fully tested my code

### CODE REVIEW

File: spec/features/visitors/visitor_can_create_an_account_spec.rb [hunk 1]
```diff
@@ -26,7 +26,7 @@
     end
 
     it 'notifies me of any info errors, and autofills my form with previous answers' do
-      expect(current_path).to eq(new_user_path)
+      expect(current_path).to eq(register_path)
```
Comment (CSheesley) [inline spec/features/visitors/visitor_can_create_an_account_spec.rb]: right now we have duplicate routes going to user#new and user#create. I have just been using the `new_user` path for most of my stuff. We just need to make a decision on which of these two routes to use and get rid of the other in our code base.

**Comment by CSheesley** [comment]
Great point about about the 10 API calls per hour for looking up a zip code - I think we should just create a locations table which stores zip, lat, long, and maybe even elevation (we just need to see what is out there). Ultimately, if we don't need to make an API call, we should avoid it.

For `user/new` and `register` routes which go to the same place - we just need to make a group decision on which way to go and keep that moving forward.

File: spec/features/visitors/visitor_can_create_an_account_spec.rb (hunk 1 above)
Comment (djc00p) [reply spec/features/visitors/visitor_can_create_an_account_spec.rb]: register

**Comment by JaxJafinPapau** [comment]
Let's go ahead and add a migration to this to drop the zip codes table from our db. No reason for it to exist anymore with the new Mapbox API call returning our locations.

**Pull request merged**